src/apps/basket/forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+
+PRODUCT_QUANTITY_CHOICES = [
```

Comment:
Why are we explicitly set max number of items we can buy?

Reply:
we can delete it, but i am not sure that this is a good choice to be able to order an endless amount of items.  Or we can leave just adding by one item

Reply:
think of other options of how we can do that
please also make limit configurable

Reply:
I see it as adding by one. On a product page you just have an option adding to the cart, for example, and in the cart you have limitation and adding just by one

Reply:
maybe we can have positive integer field with a limit?